libmscore/element.h

```diff
@@ -234,7 +235,7 @@ class Element : public QObject, public ScoreElement {
             SYSTEM_DIVIDER,
             STEM_SLASH,
             LINE,
-
+            ANNOTATION,
             ARPEGGIO,
             ACCIDENTAL,
```

Comment:
I think you'd Need to move ANNOTATION after ACCIDENTAL here, to have the same order as in element.cpp, `elementNames[]`? This is what might be causing the mtest failures?